packages/calling/src/CallHistory/CallHistory.test.ts

```diff
@@ -115,5 +127,129 @@ describe('Call history tests', () => {
 
       callSessionCallback(MOCK_SESSION_EVENT_LEGACY);
     });
+
+    it('verify the user viewed session event for missed calls update', async () => {
+      await new Promise<void>((resolve) => {
+        callHistory.on(
+          COMMON_EVENT_KEYS.CALL_HISTORY_USER_VIEWED_SESSIONS,
+          (event: CallSessionViewedEvent) => {
+            expect(event.data).toEqual(MOCK_SESSION_EVENT_VIEWED.data);
+            resolve();
+          }
+        );
+
+        expect(mockOn.mock.calls[2][0]).toEqual(MOBIUS_EVENT_KEYS.CALL_SESSION_EVENT_VIEWED);
+        const callSessionCallback = mockOn.mock.calls[2][1];
+
+        callSessionCallback(MOCK_SESSION_EVENT_VIEWED);
+      });
+    });
+  });
+
+  describe('Update missed calls test', () => {
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+    beforeEach(async () => {
+      serviceErrorCodeHandlerSpy = jest.spyOn(utils, 'serviceErrorCodeHandler');
+      global.fetch = jest.fn(() =>
+        Promise.resolve({
+          status: 200,
+          ok: true,
+          json: () => Promise.resolve(MOCK_UPDATE_MISSED_CALL_RESPONSE),
+        })
+      ) as jest.Mock;
+    });
+
+    it('successfully updates missed calls', async () => {
+      const endTimeSessionIds = [{endTime: '1234568', sessionId: '123'}];
+      const response = await callHistory.updateMissedCalls(endTimeSessionIds);
+      const convertedEndTimeSessionIds = endTimeSessionIds.map((session) => ({
+        ...session,
+        endTime: new Date(session.endTime).getTime(),
+      }));
+      expect(response.statusCode).toEqual(200);
+      expect(response).toEqual(MOCK_UPDATE_MISSED_CALL_RESPONSE);
+      expect(global.fetch).toHaveBeenCalledTimes(1);
```

Comment:
You can club these 2 expect statements into one: using toBeAclledOnceWith()

Reply:
Done